src/main/core/head/head-core.component.html

```diff
@@ -6,9 +6,9 @@
       [attr.colspan]="$column.colspan"
       [q-grid-drag]="$view.head.transfer($column.model)"
       [q-grid-drop]="$view.head.transfer($column.model)"
-      q-grid-on-drop="$view.head.drop.execute($event)"
-      q-grid-can-drop="$view.head.drop.canExecute($event)"
-      q-grid-can-drag="$view.head.drag.canExecute($event)"
+      (q-grid-on-drop)="$view.head.drop.execute($event)"
+      [q-grid-can-drop]="$view.head.drop.canExecute($column.model)"
```

Comment:
I thinks this not very good, cause it'll trigger every time on change tracker we need to pass it as a function